K2Bridge.Tests.UnitTests/KustoDAL/KustoResponseParserTests.cs

```diff
@@ -137,15 +139,17 @@ public void ParseElasticResponse_WithHits_ReturnsElasticResponseWithHits()
             Assert.AreEqual(2, elasticResult.Hits.Hits.Count());
         }
 
-        [TestCase(nameof(DateHistogramAggregation))]
-        [TestCase(nameof(TermsAggregation))]
-        public void ParseElasticResponse_WithAggs_ReturnsElasticResponseWithAggs(string primaryAggregation)
+        [Test]
+        public void ParseElasticResponse_WithDateHistogramAggs_ReturnsElasticResponseWithAggs()
```

Comment:
The terms agg test is gone?

Reply:
Yes. I created a new dedicated test for terms aggregation.